src/tss_callbacks.c

```diff
@@ -0,0 +1,118 @@
+/*
+ * This file and its contents are licensed under the Apache License 2.0.
+ * Please see the included NOTICE for copyright information and
+ * LICENSE-APACHE for a copy of the license.
+ */
+
+#include <postgres.h>
+
+#include <fmgr.h>
+#include <executor/instrument.h>
+
+#include "guc.h"
+#include "tss_callbacks.h"
+
```

Comment:
Could you add a comment to the file on why the hooks are needed (basically what is mentioned in the commit message)? This would make it easier for readers to understand the reason for the hooks without looking into the git history.